src/utils/checkUser.js

```diff
@@ -0,0 +1,20 @@
+const {users} = require("../sequelized/models")
+
+const checkUser = async (id) => {
+  try {
+    const userExist = await users.findOne({
+      where: {
+        id
+      }
```

Comment:
We can make this helper function generic as we can use `or operator` to also check for `user email`